.github/components.yml

```diff
@@ -34,41 +34,34 @@ CPU:
     - TF_FE
     - ONNX_FE
   build:
-    - HETERO
-    - AUTO_BATCH
+    - AUTO
     - TEMPLATE
     - IR_FE
 
 GPU:
   build:
-    - HETERO
```

Comment:
Why remove HETERO?

Reply:
En, if HETERO is using GPU internal header files. We should enable the build.

Reply:
GPU headers are part of `inference` component:
https://github.com/openvinotoolkit/openvino/blob/637920ce9154f47ca2c1992753406d8dbca39649/.github/labeler.yml#L85-L87
which triggers rebuild and revalidate of other dependent components:
https://github.com/openvinotoolkit/openvino/blob/637920ce9154f47ca2c1992753406d8dbca39649/.github/components.yml#L23-L24

Reply:
@ilya-lavrenov I have added AUTO/BATCH/HETERO back to the build of CPU/GPU changes, the reason is to keep same build configuration as release. They don't add too much time especially if we are using cache.